App/SB/views/Settings/index.js

```diff
@@ -126,15 +114,43 @@ class SyncPermissions extends React.PureComponent {
 }
 
 const mapStateToProps = state => {
+  // get all top level services
+  const allServices: {[index: string]: string} = Object.keys(state.preferences.services)
+    .reduce((previous, current) => {
+      let basic = state.preferences.services[current]
+      basic.info = GetServiceInfo(current)
+      previous[current] = basic
+      return previous
+    }, {})
+
+  const services: {[index: string]: string} = Object.keys(allServices)
+    .filter((key) => allServices[key].info !== undefined && allServices[key].info.dependsOn === undefined)
```

Comment:
`dependsOn` is used here too

Reply:
I moved it into the `.info` object which is only here in the UI